File: aoj/intro/ITP/ITP1_10_C.py
import math
while True:
    n = int(raw_input())
    if n == 0:
        break
    stdnt = map(int, raw_input().split())
    ave = 0
    for score in stdnt:
        ave += float(score)/n
    alpha = 0
    for score in stdnt:
        alpha += (score-ave) ** 2/n
    print math.sqrt(alpha)
    

File: aoj/intro/ITP/ITP1_10_C.c
#include <stdio.h>
#include <math.h>

int main(){
	int i,n;
	double ave,alpha,s[1001];
	while (1){
		scanf("%d",&n);
		if (n==0) break;
		ave=0.0;
		for (i=0;i<n;i++){
			scanf("%lf",&s[i]);
			ave+=s[i];
		}
		ave/=n;
		alpha=0.0;
		for (i=0;i<n;i++){
			alpha+=pow(s[i]-ave,2);
		}
		printf("%lf\n",sqrt(alpha/n));
	}
	return 0;
}

